server/game/gamepipeline.js

```diff
@@ -19,7 +19,7 @@ class GamePipeline {
     getCurrentStep() {
         var step = this.pipeline[0];
 
-        if(typeof(step) === 'function') {
+        if(typeof (step) === 'function') {
```

Comment:
Do we have eslint rules for this (if this is the formatting we want)?

Reply:
No, and I’m not even sure it is what we want, I have too many instances of vs code with differing configs.  I will sort that out in my end